.github/workflows/e2e.yaml

```diff
@@ -40,7 +42,39 @@ jobs:
           aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws_ssh_key: ${{ secrets.AWS_SSH_KEY }}
-          holodeck_config: "tests/e2e/holodeck.yaml"
+          holodeck_config: "tests/e2e/infra/aws.yaml"
+      - name: Run e2e tests
+        env:
+          KUBECONFIG: ${{ github.workspace }}/kubeconfig
+          E2E_IMAGE_REPO: ghcr.io/nvidia/k8s-device-plugin
+          E2E_IMAGE_TAG: ${COMMIT_SHORT_SHA}-ubuntu22.04
+          LOG_ARTIFACTS: ${{ github.workspace }}/e2e_logs
+        run: |
+          make -f tests/e2e/Makefile e2e-test
+      - name: Archive test logs
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: e2e-test-logs
+          path: ./e2e_logs/
+          retention-days: 15
+  nightly-e2e:
```

Comment:
Instead of doing this nightly, why not just also trigger it on push? As I mentioned, I don't see the value of nightly runs until we have better notifications of failures.

Reply:
how about now, I called it `staging` to refer as a job that is not a prod job run